From 77c3c813d059c26ea5bde37f926c8d3965d8992f Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Thu, 26 Apr 2018 15:09:38 +0800
Subject: [PATCH 1/2] arm: mach-imx: platsmp: remove unused variables

arch/arm/mach-imx/platsmp.c:44:12: warning: 'secondary_pre_boot_entry' defined but not used [-Wunused-variable]
 static u32 secondary_pre_boot_entry;
            ^~~~~~~~~~~~~~~~~~~~~~~~
arch/arm/mach-imx/platsmp.c:43:22: warning: 'scfg_base' defined but not used [-Wunused-variable]
 static void __iomem *scfg_base;
                      ^~~~~~~~~
arch/arm/mach-imx/platsmp.c:42:22: warning: 'dcfg_base' defined but not used [-Wunused-variable]
 static void __iomem *dcfg_base;

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/mach-imx/platsmp.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm/mach-imx/platsmp.c b/arch/arm/mach-imx/platsmp.c
index b1ec00b..5f344da 100644
--- a/arch/arm/mach-imx/platsmp.c
+++ b/arch/arm/mach-imx/platsmp.c
@@ -39,9 +39,6 @@
 
 u32 g_diag_reg;
 static void __iomem *scu_base;
-static void __iomem *dcfg_base;
-static void __iomem *scfg_base;
-static u32 secondary_pre_boot_entry;
  
 static struct map_desc scu_io_desc __initdata = {
 	/* .virtual and .pfn are run-time assigned */
-- 
1.8.5.2.233.g932f7e4

